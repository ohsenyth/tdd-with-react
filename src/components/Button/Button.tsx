import * as React from 'react'

const Button = () => {
    return <button></button>
}

export { Button }