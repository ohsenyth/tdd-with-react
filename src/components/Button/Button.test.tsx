import * as React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { Button } from './Button'

describe('<Button /> tests', () => {
    it('should render as default button', () => {
        const { container } = render(<Button />)
        expect(container.firstChild).toMatchSnapshot()
    })
})