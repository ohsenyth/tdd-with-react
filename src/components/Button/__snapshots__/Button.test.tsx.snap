// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> tests should render as default button 1`] = `<button />`;
